src/Main.java

```diff
@@ -1,12 +1,20 @@
 import company.InsuranceCompany;
+import exceptions.InvalidM2Exception;
+import exceptions.InvalidPriceException;
 import insurances.*;
 import people.Customer;
 import people.Employee;
 import util.Printer;
+import org.apache.logging.log4j.*;
 
 public class Main {
 
-    public static void main(String[] args) {
+    static {
+        System.setProperty("log4j.configurationFile", "log4j2.xml");
+    }
+    private static final Logger LOGGER = LogManager.getLogger(Main.class);
```

Comment:
to avoid hardcoding of classes
```suggestion
    private static final Logger LOGGER = LogManager.getLogger(MethodHandles.lookup().lookupClass());
```